zebrad/src/config.rs

```diff
@@ -174,16 +174,15 @@ impl Default for MetricsSection {
 #[derive(Clone, Debug, Deserialize, Serialize)]
 #[serde(deny_unknown_fields, default)]
 pub struct SyncSection {
-    /// The maximum number of concurrent block download requests during sync.
+    /// The number of parallel block download requests.
```

Comment:
The config changes are a good summary of the goals of this ticket - split the download, checkpoint verify, and full verify concurrency limits.